apps/docs/content/docs/components/rating.mdx

```diff
@@ -0,0 +1,175 @@
+| opacity            | `number` in `range(0, 1)`                                                                                                                                              | The opacity for the displayed icon when it is not active.                                                                                                      | `0.2`         |
+| activeOpacity      | `number` in `range(0, 1)`                                                                                                                                              | The opacity for the displayed icon when it is active.                                                                                                          | `1`           |
+| isSingleSelection  | `boolean`                                                                                                                                                              | When true only icon corresponding to rating value will be active.                                                                                              | `false`       |
+| size               | `sm` \| `md` \| `lg`                                                                                                                                                   | The size of the rating.                                                                                                                                        | `sm`          |
+| value              | `string`                                                                                                                                                               | The current value of the rating (controlled).                                                                                                                  | -             |
+| defaultValue       | `string`                                                                                                                                                               | The default value of the rating (uncontrolled).                                                                                                                | -             |
+| description        | `ReactNode`                                                                                                                                                            | A description for the rating. Provides a hint such as specific requirements for what to choose.                                                                | -             |
+| errorMessage       | `ReactNode`                                                                                                                                                            | A errorMessage for the rating.                                                                                                                                 | -             |
```

Comment:
The `errorMessage` prop can also accept a function. Please refer to the [Radio docs](https://nextui.org/docs/components/radio-group#radiogroup-props).
Additionally, please add a story for it.